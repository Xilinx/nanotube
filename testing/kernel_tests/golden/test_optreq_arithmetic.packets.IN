###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
# Convert with: text2pcap -t '%Y-%m-%d %H:%M:%S.' <in-file> <out-file>

# Offset 0, sel0 0, sel1 0
1980-10-12 17:40:00.000000
0000 00 00 00 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 1, sel1 0
1980-10-12 17:40:00.000000
0000 00 00 01 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 2, sel1 0
1980-10-12 17:40:00.000000
0000 00 00 02 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 3, sel1 0
1980-10-12 17:40:00.000000
0000 00 00 03 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 0, sel1 1
1980-10-12 17:40:00.000000
0000 00 00 00 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 1, sel1 1
1980-10-12 17:40:00.000000
0000 00 00 01 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 2, sel1 1
1980-10-12 17:40:00.000000
0000 00 00 02 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 0, sel0 3, sel1 1
1980-10-12 17:40:00.000000
0000 00 00 03 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 0, sel1 0
1980-10-12 17:40:00.000000
0000 03 00 00 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 1, sel1 0
1980-10-12 17:40:00.000000
0000 03 00 01 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 2, sel1 0
1980-10-12 17:40:00.000000
0000 03 00 02 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 3, sel1 0
1980-10-12 17:40:00.000000
0000 03 00 03 00 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 0, sel1 1
1980-10-12 17:40:00.000000
0000 03 00 00 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 1, sel1 1
1980-10-12 17:40:00.000000
0000 03 00 01 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 2, sel1 1
1980-10-12 17:40:00.000000
0000 03 00 02 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f

# Offset 3, sel0 3, sel1 1
1980-10-12 17:40:00.000000
0000 03 00 03 01 01 02 03 04 05 06 07 08 09 0a 0b 0c
0010 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020 1d 1e 1f
