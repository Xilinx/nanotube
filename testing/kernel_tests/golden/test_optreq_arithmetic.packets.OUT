###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
1970-01-01 00:00:00.000000
0000  05 04 03 02 07 08 0a 0b 78 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  04 03 02 01 06 07 09 0a 70 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  02 03 04 05 06 07 09 0a 70 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  01 02 03 04 05 06 08 09 68 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  05 04 03 02 07 08 09 0a 70 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  04 03 02 01 06 07 08 09 68 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  02 03 04 05 06 07 08 09 68 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  01 02 03 04 05 06 07 08 60 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  05 04 03 02 07 08 0d 0e 90 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  04 03 02 01 06 07 0c 0d 88 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  02 03 04 05 06 07 0c 0d 88 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  01 02 03 04 05 06 0b 0c 80 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  05 04 03 02 07 08 0c 0d 88 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  04 03 02 01 06 07 0b 0c 80 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  02 03 04 05 06 07 0b 0c 80 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
1970-01-01 00:00:00.000000
0000  01 02 03 04 05 06 0a 0b 78 00 00 00 09 0a 0b 0c
0010  0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c
0020  1d 1e 1f
